### ISSUE TITLE
Option to skip external mod downloads

### ISSUE THREAD

**Issue by stealingudc** [killpowa/Twickt-Launcher#1577]
- [x] I searched the issue tracker for other issues covering my case.

**Is your feature request related to a problem? Please describe.**
When downloading a CurseForge modpack, after the 'unofficial downloads' popup, the browser attempts to download the required mods. On most occasions, this works perfectly, however, with certain mods (i.e. Iris Shaders at the time of submitting), after the CurseForge ad countdown, the browser shows a 404 request code being returned. As a consequence, I can not play the modpack at all, since my options are either to retry the download or delete the instance.

**Describe the solution you'd like**
Implementing a check for whether or not an API call returns 404, 'pausing' the download process and giving the user the option to skip downloading the mod, *warning them about the pack potentially breaking*.

**Describe alternatives you've considered**
Downloading the pack through CurseForge, and importing it in GDL as .zip. This is not exactly an option for Linux users, since many modpacks don't appear on their website, but do appear in both their launcher and GDL (i.e. All Of Fabric 6)

**Comment by trevorjd** [comment]
Agreed.  I ran into this tonight.  It has downloaded everything except Sodium and is giving a 404 for that one.  It's utterly absurd that the only options I have are to try again (4th time, still 404) or delete the entire instance!  We should have a third option, "Leave it as-is and I'll deal with it myself."

(edit: Sodium has departed Curseforge completely.  You might also want to have the launcher check Modrinth if it can't find things on CF)

**Comment by RandomlyZay** [comment]
Litterally just encountered this issue and came here to see if there was any workaround, but it doesn't look like there is one. Was trying to update a modpack and one of the mods (Sodium) can't be downloaded on Curseforge anymore so I don't know what to do. Having an "ignore/skip failed downloads" button would be a godsend for these kinds of issues.

**Comment by trevorjd** [comment]
I downloaded and installed Prism launcher.  It gives you a list of what it needs and auto-watches c:\downloads so you can click the links, download to \downloads and it'll sit and wait until you manually fetch what it needs.

**Comment by Poopooracoocoo** [comment]
GDLauncher doesn't know what to do with archived mods. An update to fix that seems unlikely to me as I think they're focused on the rewrite that wouldn't have this problem at all because it uses CurseForge's partner APIs. 

As a workaround, you can just close GDLauncher. You can then open it again and it will keep your install and skip the mods it couldn't download. 

You can then right click on the instance and select open folder and then add downloaded mods from another source like Modrinth to the mods folder. Make sure it's the same version. 

Also see #1575
